addons/mail/static/src/core/common/composer.js

```diff
@@ -206,6 +202,14 @@ export class Composer extends Component {
         });
     }
 
+    prepareStateParameters() {
+        return {
+            autofocus: 0,
+            active: true,
+            keyboard: this.KEYBOARD.NONE,
+        };
+    }
+
```

Comment:
I'd introduce this function for patching, as it's poorly named and makes the code harder to read.

Instead, I would just patch `setup()` with `this.state.recording = false`